autokeras/blocks/basic.py

```diff
@@ -974,14 +976,11 @@ def from_config(cls, config):
     def build(self, hp, inputs=None):
         input_tensor = nest.flatten(inputs)[0]
 
-        tokenizer_layer = keras_layers.BertTokenizer(
-            max_sequence_length=utils.add_to_hp(self.max_sequence_length, hp)
-        )
-        output_node = tokenizer_layer(input_tensor)
-
-        bert_encoder = keras_layers.BertEncoder()
+        preset_name = "bert_base_en_uncased"
```

Comment:
Depending on your application and user base, does using an EN-only model make sense? We offer `"bert_base_multi_cased"` which is trained on 104 languages as well as much more powerful models such as `XLMRoberta` which have training data and tokenization optimized for multilingual settings.

Reply:
For this PR, we try to keep it consistent as before. So we chose this preset.

The multi-language model is a great idea! In the future, we plan to let the user specify their language for `TextClassifier`, base on which AutoKeras will choose the right model and preset for them.

Reply:
Sure, happy to talk later! After BERT, most models were trained on a unified multilingual corpus. In fact, XLMRoberta is famous for showing that this is even better for EN-only. So with the right model you may not need to do this.